sql/moz-fx-data-shared-prod/mozilla_vpn_derived/funnel_product_page_to_subscribed_v1/query.sql

```diff
@@ -206,7 +206,12 @@ SELECT
   pricing_plan,
   plan_name,
   rp_button_view AS vpn_site_hits,
-  mozfun.vpn.channel_group(utm_medium, utm_source, utm_campaign, utm_content) AS channel_group,
+  mozfun.vpn.channel_group(
+    utm_campaign => utm_campaign,
+    utm_content => utm_content,
+    utm_medium => utm_medium,
+    utm_source => utm_source
+  ) AS channel_group,
```

Comment:
arguments were out of order, causing channel group to be incorrectly assigned

Reply:
I'd suggest changing the `channel_group()` call in `channel_group_proportions_v1` to used named parameters as well for consistency.